### ISSUE TITLE
Not able to deploy dtrack.war which is generated from source

### ISSUE THREAD

**Issue by kuldipdp** [stevespringett/dependency-track#38]
Hi Steve,

I have followed steps which is mention on following link:

https://github.com/stevespringett/dependency-track/wiki/Getting-Started

1) downloaded src
2) did maven clean package
3) and try to deploy dtrack.war generated in target folder.

But i am facing issue. PFA of error log
[error log.txt](https://github.com/stevespringett/dependency-track/files/512886/error.log.txt)

while deploying war file given at same place, it is getting deployed successfully.

Looking at the exception, it is pointing with JDK vresion competibility issue, But I have also check the version of class file from war file which is generated from war. which is jdk 7+ so, i dont feel that can be a issue

I have also compare both the war file and found that web.xml is not getting generated in src war. and copied same at in web-inf folder, and generated dtrack.war again. and tried to deploy again but still facing same issue.

could you kindly help me.

**Comment by joshijimit** [comment]
There is nothing wrong with what mentioned https://github.com/stevespringett/dependency-track to set it up.. Just run below steps after cloning code in local.

1) mvn clean package
2) java- jar target/dtrack.war
3) http://localhost:8080/dtrack - you will see this page.. Default credentials is admin/admin which too is working fine..

**Comment by kuldipdp** [comment]
That's correct I am also able to do that way but when i am trying from tomcat after downloading source and setting up in local and generating war and trying yo deploying same.

appreciate your help.

**Comment by stevespringett** [comment]
FYI, I have branched the project to make the source tree more clear.

- The only stable version (which can also be downloaded in a ready-to-deploy war) is 1.0.x.
- The master and 1.0-stable branches are in sync and should provide an easy way to compile your own war resulting in a working application.
- The 2.0-dev-springboot branch is unstable and has many issues. I do not intend to ever release this version. Just too many issues overall. This work was previously performed in the master branch but has since been reverted and separated to its own branch.
- All new development is being done in the 3.0-dev branch and is a complete rewrite of Dependency-Track from the ground up using modern technologies and an API-first design. It also doesn't rely on Spring.
